### DISCUSSION TITLE
Transfer function to internal nodes using Circuit

### DISCUSSION THREAD

**Discussion by tzortea** [scikit-rf/scikit-rf#1157] [Q&A] [answered]
Is it possible to get a transfer function (not just s-parameters) from an external port to an internal node (one that is not connected to an external port) from a `Circuit `object? This transfer function should account for the loading that the internal node sees from it's connections to other nodes in the `Circuit`.

Is this what `Circuit.voltages()` gives? Say you set one port to have power=1,phase=0 and all other ports power=0, then from `Circuit.voltages(pow,ph)` you have the transfer function from that one port to all internal nodes?

**Comment by jhillairet** [comment, marked as answer]
I'm not sure to really understand what you mean with the "transfert function". 

`voltages()` gives the voltages at each nodes of the circuit for a given external port power/phase excitation. For example, if you have two external ports, `.voltages([1, 2], [0, 0.2])` will give you all the voltages at the internal port of your circuit when putting 1 Watt/0rad at port 1 and 2 Watt/0.2 rad at port 2.
